com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PrepareAppsRestartStep.java

```diff
@@ -22,6 +24,8 @@ protected StepPhase executeStep(ExecutionWrapper execution) {
             .setVariable(Constants.EXECUTE_ONE_OFF_TASKS, false);
         StepsUtil.setUseIdleUris(execution.getContext(), false);
         StepsUtil.setSkipUpdateConfigurationEntries(execution.getContext(), false);
+        
+        StepsUtil.setIteratedAppsInParallel(execution.getContext(), Collections.emptyList());
```

Comment:
Why is this still here? You should be able to remove it after providing a default value in the getter.

Reply:
Read the previous comment on this. This is not a default value. It is there to "reset" the iteration from the deployment operation in order for restart call activity to reiterate apps. Getter would return default value only if there was nothing in the context. In this case there is (all apps) and we need to reset that.